src/ots.h

```diff
@@ -209,13 +206,19 @@ bool IsValidVersionTag(uint32_t tag);
 #define OTS_TAG_OS2  OTS_TAG('O','S','/','2')
 #define OTS_TAG_POST OTS_TAG('p','o','s','t')
 #define OTS_TAG_PREP OTS_TAG('p','r','e','p')
-#define OTS_TAG_SILE OTS_TAG('S','i','l','e')
-#define OTS_TAG_SILF OTS_TAG('S','i','l','f')
-#define OTS_TAG_SILL OTS_TAG('S','i','l','l')
 #define OTS_TAG_VDMX OTS_TAG('V','D','M','X')
 #define OTS_TAG_VHEA OTS_TAG('v','h','e','a')
 #define OTS_TAG_VMTX OTS_TAG('v','m','t','x')
 #define OTS_TAG_VORG OTS_TAG('V','O','R','G')
+// Graphite tables
+#ifdef OTS_GRAPHITE
```

Comment:
This does not need to be behind the `#ifdef`, we better always have the macro around just in case it is needed.